test/test_cpp_api_parity.py

```diff
@@ -275,10 +315,14 @@ def generate_attr_equality_checks(module,
                         script_module_prefix=script_module_prefix,
                         cpp_module_prefix=cpp_module_prefix,
                         buffer_name=name))
-                module_metadata = torch_nn_modules.module_metadata_map[module.__class__.__name__]
+
+                init_arg_spec = self._get_python_module_init_arg_spec(module.__class__.__name__)
+                python_constructor_arg_names = [x for x in init_arg_spec.args[1:] if x != 'has_parity']
```

Comment:
Why does this exclude argument 0 / what is argument 0? Might be good to write a comment here.